marlowe-playground-client/psc-package.json

```diff
@@ -21,6 +21,6 @@
         , "simple-json"
         , "psci-support"
         ],
-    "name": "meadow-client",
+    "name": "marlowe-playground-client",
```

Comment:
hm, I'd expect the nix stuff to need regenerating because of this?

Reply:
I built and deployed on my environment, it worked